common/iso_config.go

```diff
@@ -81,6 +82,21 @@ func (c *ISOConfig) Prepare(ctx *interpolate.Context) ([]string, []error) {
 							return warnings, errs
 						}
 					case "file":
+						// Windows file handling is all sorts of tricky...
+						if runtime.GOOS == "windows" {
```

Comment:
Not sure if the [std lib](https://golang.org/pkg/path/filepath/) will help. Yes I completely agree it's better to go for a more generic approach. But due to my lack of experience, I re-used the code snippet from [config.go](https://github.com/mitchellh/packer/blob/master/common/config.go#L58-L73) to get the test to pass again.